internal/node/node_repositories.bzl

```diff
@@ -591,11 +591,7 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 
     # The entry points for yarn for osx/linux and windows.
     # Runs yarn using appropriate node entry point.
-    # Unset YARN_IGNORE_PATH before calling yarn incase it is set so that
-    # .yarnrc yarn-path is followed if set. This is for the case when calling
-    # bazel from yarn with `yarn bazel ...` and yarn follows yarn-path in
-    # .yarnrc it will set YARN_IGNORE_PATH=1 which will prevent the bazel
-    # call into yarn from also following the yarn-path as desired.
+    # Set YARN_IGNORE_PATH=1 so it doesn't go throug our .yarn/releases/yarn.sh wrapper
```

Comment:
nit: `throug` -> `through`
Should also generalise comment.

Reply:
```suggestion
    # Set YARN_IGNORE_PATH=1 so it doesn't go through wrappers set by `yarn-path` https://classic.yarnpkg.com/lang/en/docs/yarnrc/#toc-yarn-path
```